### PR TITLE
Some SecretManager changes

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: client/bindings/nodejs/test/client/offlineSigningExamples.spec.ts [hunk 1]
```diff
@@ -97,8 +97,8 @@ describe('Offline signing examples', () => {
 
         const unlock = await offlineClient.signatureUnlock(
             secretManager,
-            preparedTx.inputsData[0],
             hexToBytes(txEssenceHash),
+            preparedTx.inputsData[0].chain!,
```
Comment (thibault-martinez) [inline client/bindings/nodejs/test/client/offlineSigningExamples.spec.ts:101]: I'm not too sure about this...

Comment (Thoralf-M) [reply client/bindings/nodejs/test/client/offlineSigningExamples.spec.ts:101]: What's the `!` for?

Comment (thibault-martinez) [reply client/bindings/nodejs/test/client/offlineSigningExamples.spec.ts:101]: To get the value out of the optional

Comment (thibault-martinez) [reply client/bindings/nodejs/test/client/offlineSigningExamples.spec.ts:101]: Basically an unwrap

**Pull request closed without merging**